uwblueprint/planet-read
# Implement base User Profile Page

## Notion ticket link



[Set up UserProfilePage.tsx](https://www.notion.so/uwblueprintexecs/Set-up-UserProfilePage-tsx-d99f4a76842948b3b79adf28dcb23015)
[Delete user on UserProfilePage](https://www.notion.so/Delete-user-on-UserProfilePage-84e092fc394547f7937420a57a4867ff)



## Implementation description

- implemented sidebar with headings
- implemented main page headers (no tables)
- implemented footer (non-functioning buttons)
- delete button is functional
- page layout is similar to: https://github.com/uwblueprint/planet-read/pull/154



## Steps to test

1. Login as Angela merkel
2. Go to http://localhost:3000/user/1
3. Modify the number to see different users
4. Attempt to delete user
5. Try to open the url again, verify 404 page



## What should reviewers focus on?

- does the delete button work?
- does the spacing and layout look good?
- is the role displayed properly? (carl sagan should be Translator only)

## Checklist
